<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>options</class>
 <widget class="QWidget" name="options">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>324</width>
    <height>333</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTableWidget" name="lstSongs">
       <property name="showDropIndicator" stdset="0">
        <bool>false</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="showGrid">
        <bool>false</bool>
       </property>
       <property name="gridStyle">
        <enum>Qt::SolidLine</enum>
       </property>
       <property name="columnCount">
        <number>3</number>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>ID</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Song</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Artist</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="cmdAddSong">
         <property name="text">
          <string>Add song</string>
         </property>
         <property name="icon">
          <iconset resource="../core/resources.qrc">
           <normaloff>:/main/icons/add-16.png</normaloff>:/main/icons/add-16.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cmdEditSong">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Edit song</string>
         </property>
         <property name="icon">
          <iconset resource="../core/resources.qrc">
           <normaloff>:/main/icons/edit-16.png</normaloff>:/main/icons/edit-16.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cmdDeleteSong">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Delete song</string>
         </property>
         <property name="icon">
          <iconset resource="../core/resources.qrc">
           <normaloff>:/main/icons/delete-16.png</normaloff>:/main/icons/delete-16.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../core/resources.qrc"/>
 </resources>
 <connections/>
</ui>
